### DISCUSSION TITLE
BrowserStack icon

### DISCUSSION THREAD

**Discussion by mrleblanc101** [elrumo/macOS_Big_Sur_icons_replacements#1521] [Icon request] [answered]
Could someone make an icon for the BrowserStack local app ?

**Comment by vikasbhatia** [comment, marked as answer]
Here are the PNGs you can use right away:

<img width="256" alt="1" src="https://github.com/user-attachments/assets/d5a0e61d-524c-4947-8aec-1d154dbc2b07" />
<img width="256" alt="2" src="https://github.com/user-attachments/assets/c64857f8-5c55-4a6d-97e4-5df621211809" />
<img width="256" alt="3" src="https://github.com/user-attachments/assets/7b389e4d-d511-43c3-8740-fddd014203fd" />

You'll be able to download the ICNS from my profile once it's approved: https://macosicons.com/#/u/sidewinder
